### PR TITLE
docs: refactor examples tags

### PR DESCRIPTION
**Pull request by yildirayunlu**
Update examples and refactor tags.

### Self Check before Merge

Please check all items below before review.

-   [x] Corresponding issues are created/updated or not needed
-   [x] Docs are updated/provided or not needed
-   [x] Examples are updated/provided or not needed
-   [x] TypeScript definitions are updated/provided or not needed
-   [x] Tests are updated/provided or not needed
-   [x] Changesets are provided or not needed

### CODE REVIEW

File: documentation/docs/examples/authentication/chakra-ui.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 id: chakra-ui
 title: Chakra UI
 example-title: Authentication with Chakra UI
-example-tags: [chakra-ui,authentication,vite,react-table]
+example-tags: [chakra-ui,authentication,react-table]
```
Comment (omeraplak) [inline documentation/docs/examples/authentication/chakra-ui.md:5]:
```suggestion
example-tags: [chakra-ui,authentication]
```

File: documentation/docs/examples/authentication/headless.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 id: headless
 title: Headless
 example-title: Authentication Example
-example-tags: [headless,authentication,vite,react-hook-form,react-table]
+example-tags: [headless,authentication,react-hook-form,react-table]
```
Comment (omeraplak) [inline documentation/docs/examples/authentication/headless.md:5]:
```suggestion
example-tags: [headless,authentication]
```

File: documentation/docs/examples/authentication/mantine.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 id: mantine
 title: Mantine
 example-title: Authentication with Mantine
-example-tags: [mantine,authentication,vite,react-table]
+example-tags: [mantine,authentication,react-table]
```
Comment (omeraplak) [inline documentation/docs/examples/authentication/mantine.md:5]:
```suggestion
example-tags: [mantine,authentication]
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

File: documentation/docs/examples/table/handsontable/basic.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 id: handsontable
 title: Basic
 example-title: Handsontable Example
-example-tags: [table,handsontable,vite]
+example-tags: [table,handsontable]
```
Comment (omeraplak) [inline documentation/docs/examples/table/handsontable/basic.md:5]:
```suggestion
example-tags: [table]
```

File: documentation/docs/examples/themes/refine-themes-chakra-ui.md [hunk 5]
```diff
@@ -2,7 +2,7 @@
 id: refine-themes-chakra-ui
 title: Chakra UI
 example-title: Chakra UI
-example-tags: [chakra-ui, customization, theme]
+example-tags: [chakra-ui,customization,theme]
```
Comment (omeraplak) [inline documentation/docs/examples/themes/refine-themes-chakra-ui.md:5]:
```suggestion
example-tags: [chakra-ui,customization]
```

**Review by omeraplak — CHANGES_REQUESTED** [review]

File: documentation/docs/examples/web3/signin-ethereum.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
 id: web3Login
 title: Sign-In with Ethereum
-example-tags: [web3,auth-provider,antd]
+example-tags: [web3,auth-provider]
```
Comment (omeraplak) [inline documentation/docs/examples/web3/signin-ethereum.md:4]:
```suggestion
example-tags: [auth-provider]
```

File: documentation/docs/examples/audit-log/audit-log-antd.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+---
+id: audit-log-antd
+title: Audit Log with Antd
+example-tags: [audit-log,antd]
```
Comment (BatuhanW) [inline documentation/docs/examples/audit-log/audit-log-antd.md:4]: Do we need antd here? Audit log may not be specific to antd.

File: documentation/docs/examples/base/mui-base.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+---
+slug: mui-base
+id: mui-base
+title: Material UI Base
+example-title: Material UI
```
Comment (BatuhanW) [inline documentation/docs/examples/base/mui-base.md:5]:
```suggestion
title: Material UI
example-title: Material UI Base
```

File: documentation/docs/examples/import-export/antd.md [hunk 9]
```diff
@@ -1,8 +1,8 @@
 ---
-id: import-export
-title: Import-Export
-example-title: Import-Export with Ant Design
-example-tags: [antd,import,export,refine-hooks,data-provider]
+id: antd
+title: Ant Design
+example-title: Ant Design
+example-tags: [import,export]
```
Comment (BatuhanW) [inline documentation/docs/examples/import-export/antd.md:5]: We have similar example for useImport says headless, maybe we can use another tag for both, like csv? import. export isn't clear enough on what's being imported.

**Review by BatuhanW — COMMENTED** [review]
Added some comments.

File: documentation/docs/examples/audit-log/audit-log-antd.md (hunk 7 above)
Comment (yildirayunlu) [reply documentation/docs/examples/audit-log/audit-log-antd.md:4]: We have an [`antd-audit-log`](https://github.com/refinedev/refine/tree/next/packages/antd-audit-log) package. We used this example.

**Review by BatuhanW — APPROVED** [review]

File: documentation/src/components/example-list/index.tsx [hunk 10]
```diff
@@ -46,7 +42,6 @@ const visibleTags = staticTags.map((tag) => ({
 
 const PREDEFINED_COLORS = {
     react: "#61dafb",
-    vite: "#646cff",
```
[the remark this answers is not in the document]
Comment (yildirayunlu) [reply documentation/src/components/example-list/index.tsx:49]: No, because many of our examples are Vite.

**Review by omeraplak — APPROVED** [review]

**Review by aliemir — APPROVED** [review]

**Pull request merged**